src/googleads/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
need to fix linter - please run `npm run projen` and fix errors once you address remaining comments